include/utils/parameters.h

```diff
@@ -40,7 +40,7 @@
 namespace libMesh
 {
 /**
- * Helper functions for printing scalar and vector types.  Called from Parameters::Parameter<T>::print(...).
+ * Helper functions for printing scalar, vector and double vector types.  Called from Parameters::Parameter<T>::print(...).
```

Comment:
I got confused by the "double" terminology in this comment and the comment below, until I realized you meant "2D array".  Other than that, the code looks fine.